src/components/book/DeleteBook.js

```diff
@@ -82,8 +81,8 @@ class DeleteScream extends Component {
               </DialogTitle>
               <DialogContent className={classes.dialogContent}>
                 <p>
-                  Are you sure you want to{" "}
-                  <span>delete {this.props.title}</span> ?
+                  Are you sure you want to delete{" "}
```

Comment:
do you need the {""}?

Reply:
If <span>delete {this.props.title}</span> ? is positioned on same line with  'Are you sure you want to{" "}' , {" "} is not necessary. If each is separated in each line, we need it.

Reply:
ohhh ok I didn't know! :)